benchmarks/datasets.py

```diff
@@ -0,0 +1,30 @@
+import warnings
+
+import torchtext
+from torchtext.data.utils import get_tokenizer
+
+
+def get_wikitext2_data(device):
+    with warnings.catch_warnings(record=True) as _:
+        TEXT = torchtext.data.Field(
+            tokenize=get_tokenizer("basic_english"), init_token="<sos>", eos_token="<eos>", lower=True
+        )
+        train_txt, val_txt, test_txt = torchtext.datasets.WikiText2.splits(TEXT)
+        TEXT.build_vocab(train_txt)
+        ntokens = len(TEXT.vocab.stoi)
+
+        batch_size = 20
+        eval_batch_size = 10
+        train_data = batchify(train_txt, batch_size, TEXT, device)
+        val_data = batchify(val_txt, eval_batch_size, TEXT, device)
+        test_data = batchify(test_txt, eval_batch_size, TEXT, device)
+
+        return ntokens, train_data, val_data, test_data
+
+
+def batchify(data, bsz, TEXT, device):
```

Comment:
add some docstring for these two functions will help new readers who are not familiar with NLP.

Reply:
All CAP TEXT argument name looks strange to me.

Reply:
Done. Removed the CAP TEXT args, not sure why they were in CAPs.